app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -124,3145 +129,3064 @@ kong_version_compatibility:
+    - group: HTTP Client Proxy Settings
+      description: Parameters only needed if the HTTP(S) requests to identity provider need to go through a proxy server.
+    - name: http_proxy
+      required: false
+      default: 
+      datatype: url
+      description: The HTTP proxy
+    - name: http_proxy_authorization
+      required: false
+      default: 
+      datatype: string
+      description: The HTTP proxy authorization.      
+    - name: https_proxy
+      required: false
+      default: 
+      datatype: url
+      description: The HTTPS proxy
+    - name: https_proxy_authorization
+      required: false
+      default: 
+      datatype: string
+      description: The HTTPS proxy authorization.      
+    - name: no_proxy
+      required: false
+      default: 
+      datatype: array of string elements
+      description: Do not use proxy with these hosts.
+    - group: Cache TTLs
+    - name: cache_ttl
+      required: false
+      default: 3600
+      datatype: integer
+      description: The default cache ttl in seconds that is used in case the cached object does not specify the expiry.
+    - name: cache_ttl_max
+      required: false
+      default: 
+      datatype: integer
+      description: The maximum cache ttl in seconds (enforced).
+    - name: cache_ttl_min
+      required: false
+      default: 
+      datatype: integer
+      description: The minimum cache ttl in seconds (enforced).
+    - name: cache_ttl_neg
+      required: false
+      default: (derived from Kong configuration)
+      datatype: integer
+      description: The negative cache ttl in seconds.
+    - name: cache_ttl_resurrect
+      required: false
+      default: (derived from Kong configuration)
+      datatype: integer
+      description: The resurrection ttl in seconds.
+    - group: Cache Settings for the Endpoints
+    - name: cache_tokens
+      required: false
+      default: true
+      datatype: boolean
+      description: Cache the token endpoint requests.
+    - name: cache_tokens_salt
+      required: false
+      default: (auto generated)
+      datatype: string
+      description: |
+        Salt used for generating the cache key that us used for caching the token
+        endpoint requests.
+        > If you use multiple plugin instances of the OpenID Connect
+        > plugin and want to share token endpoint caches between the plugin
+        > instances, set the salt to the same value on each plugin instance.                      
+    - name: cache_introspection
+      required: false
+      default: true
+      datatype: boolean
+      description: Cache the introspection endpoint requests.         
+    - name: cache_token_exchange
+      required: false
+      default: true
+      datatype: boolean
+      description: Cache the token exchange endpoint requests.         
+    - name: cache_user_info
+      required: false
+      default: true
+      datatype: boolean
+      description: Cache the user info requests.
+  extra: |
```

Comment:
The way this publishes - it is not clear to me what it is referencing - once what is applied??

Reply:
@HCloward do you mean:
> Cache the user info requests.

So this boolean is an option to turn off (or on) the caching of user info requests that plugin can make. User info endpoint is located on idp where the request is made. So when the request is cached, we do not call it again with same token because we can find it in cache.

Should it be:
`Whether to cache the user info requests?`

Reply:
I meant the text in the Extra field: "Once applied, any user with a valid credential can access the Service.
    To restrict usage to only some authenticated users, you can use authorization
    features of the plugin, or you can integrate with the [ACL](/plugins/acl/) plugin
    (not covered here) using `config.authenticated_groups_claim`." Once published it appears in an awkward location that makes it unclear what is being referenced.

Reply:
@HCloward ok, i will remove it. It was just that many auth plugins have this exact `extra`. But sure, it is unneccessary.